CONTRIBUTING.md

```diff
@@ -0,0 +1,121 @@
+- Bugs: The title should be a declarative statement of how things are. **"Text does not blink"**
+
+## Pull Request Guidelines
+
+Pull requests must include code documentation, tests, follow code style and commits format.
+
+### Documentation
+
+All methods, functions and object structures should be documented following [JSDoc](https://jsdoc.app/index.html) style comments.
+
+### Testing
+
+We take testing very seriously, all code changes must include unit, integration and end-to-end tests.
+
+- **Unit**: Tests at the file level with mocked external requests
+- **Integration**: Tests at the application level with mocked I/O requests
+- **End-to-end**: Tests the application in a system
+
+### Code Style
+
+Code style is enforced by [linters](https://eslint.org). Use `npm run test:eslint` to verify that your code is beautiful, too!
+We highly discourage disabling eslint rules.
+Unless there is an exceptional use case, we may request additional changes to your PR.
+
+### Git Commit
+
+As part of the build process, commits are run through [conventional changelog](https://github.com/conventional-changelog/conventional-changelog) to generate the changelog. Please adhere to the following guidelines when formatting your commit messages.
+
+#### Commit Message Format
+
+Each commit message consists of a **header**, a **body** and a **footer**. The header has a special format that includes a **type**, a **scope** and a **subject**:
+
+    <type>(<scope>): <subject>
+    <BLANK LINE>
+    <body>
+    <BLANK LINE>
+    <footer>
+
+The **header** is mandatory and the scope of the header is optional.
+
+Any line of the commit message cannot be longer 79 characters! This allows the message to be easier to read on GitHub as well as in various git tools.
+
+#### Revert
+
+If the commit reverts a previous commit, it should begin with `revert:`, followed by the header of the reverted commit.
+In the body it should say: `This reverts commit <hash>`., where the hash is the SHA of the commit being reverted.
+The body should also explain why the commit was reverted.
+
+#### Type
+
+Must be one of the following:
+
+- **build**: Changes that affect the build system or external dependencies (example scopes: gulp, broccoli, npm)
+- **ci**: Changes to our CI configuration files and scripts (example scopes: Circle, BrowserStack, SauceLabs)
+- **docs**: Documentation only changes
+- **feat**: A new feature
+- **fix**: A bug fix
+- **perf**: A code change that improves performance
+- **refactor**: A code change that neither fixes a bug nor adds a feature
+- **style**: Changes that do not affect the meaning of the code (white-space, formatting, missing semi-colons, etc)
+- **test**: Adding missing tests or correcting existing tests
+
+#### Scope
+
+The scope should indicate what is being changed. Generally, these should match widgets names. For example, `WebexWidgets`, etc. Other than those, `tooling` tends to be the most common.
```

Comment:
```suggestion
The scope should indicate what is being changed. Generally, these should match widgets names. For example, `WebexSpaceWidget`, etc. Other than those, `tooling` tends to be the most common.
```

Reply:
Should we do `WebexMeetingWidget` to start off with, since that is the only thing in this repo right now?

Reply:
Sure